### PR TITLE
update the area of focus template to match the new model fields

### LINKED ISSUE #4935: [Frontend] Areas of Focus
Desktop:

Screen Shot 2020-07-29 at 9 50 41 AM

Tablet:

Screen Shot 2020-07-29 at 9 51 40 AM

Mobile:

Screen Shot 2020-07-29 at 9 52 07 AM

### PR DESCRIPTION
**Pull request by Pomax**
Closes #4935 by updating the already written template to use the updated mode fields - it was still pointing at the old field name, so nothing was showing up

live url: https://foundation-s-ia-refresh-1bzb4x.herokuapp.com/en/
edit url: https://foundation-s-ia-refresh-1bzb4x.herokuapp.com/cms/pages/3/edit/

### CODE REVIEW

File: network-api/networkapi/wagtailpages/factory/__init__.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 from . import (
-    focus_areas,
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/factory/__init__.py:2]: this doesn't need a factory, the default areas get built as part of the standard migrations already.

File: network-api/networkapi/wagtailpages/pagemodels/base.py [hunk 2]
```diff
@@ -516,6 +516,19 @@ class Meta:
         verbose_name_plural = 'Areas of focus'
 
 
+class HomepageFocusAreas(WagtailOrderable):
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/pagemodels/base.py:519]: moved this up so it's next to the FocusArea class.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/focus_area.html [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
     <div class="focus-content">
         <h2 class="h3-heading">{{ area.name }}</h2>
         <p>{{ area.description }}</p>
 
-        {# The following code is based on the requirements noted in https://github.com/mozilla/foundation.mozilla.org/issues/4989 #}
-
-        {% if area.page %}
-            <a href="{{ area.page.url }}">{% trans "Learn more →" %}</a>
-        {% endif %}
+        {% if area.page %}<a href="{{ area.page.url }}">{% trans "Learn more →" %}</a>{% endif %}
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/focus_area.html:9]:
```suggestion
        {% if area.page %}<a href="{{ area.page.url }}" class="cta-link">{% trans "Learn more" %}</a>{% endif %}
```

We can also get this updated in the frontend PR for Focus Areas

Comment (Pomax) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/focus_area.html]: the cta-link class is different than what's requested (18px/bold), so I'll use a custom rule in home.scss

**Comment by Pomax** [comment]
let's see if bootstrap easily allows that.

**Pull request merged**